test/e2e/service_bootstrap.py

```diff
@@ -33,7 +35,13 @@ def service_bootstrap() -> Resources:
             aws_service_name="opensearchservice.amazonaws.com",
             default_name="AWSServiceRoleForAmazonOpenSearchService",
             description="An SLR to allow Amazon OpenSearch Service to work within a private VPC"
-        )
+        ),
+        MasterUserPasswordSecret=secret.Secret(
+            ns="default",
+            name="passwords",
+            key="master_user_pass",
+            val="Secretpass123456!",
```

Comment:
FWIW, opensearch requires the password contain a number, an uppercase, a lowercase, and a special character...